models/models.js

```diff
@@ -77,4 +77,18 @@ exports.updateReviewById = (reviewId, updateInformation) => {
         .then(({rows}) => {
             return rows[0]
         })
-}
\ No newline at end of file
+}
+
+exports.selectCommentsByReviewId = (reviewId) => {
+    return db.query(`SELECT * FROM reviews WHERE review_id = $1`, [reviewId])
+        .then(({rows}) => {
+            const reviews = rows
+            if (reviews.length === 0) return Promise.reject({status: 404, msg: `Review ID: ${reviewId} does not exist`})
+            
+            return db.query(`SELECT * FROM comments WHERE review_id = $1`, [reviewId])
+        })
+        .then(({rows}) => {
+            const comments = rows
+            return comments
+        })
```

Comment:
no need to reassign variables here

just return comments!

if you really wanted to rename it you can do so like this:
```js
.then(({ rows: comments }) => {
```